### PR TITLE
Notgithub: a service to simulate GitHub token scanning

### PR DESCRIPTION
**Pull request by ewjoachim**
Because manually testing the GitHub token scanning code in my dev environment was nontrivial to say the least, I've made a service to automate the complicated parts, and here's the integration of this service to the Warehouse dev environment.

Also, adds token scanning doc!

Capture d’écran 2021-03-20 à 18 40 50

Ping @greysteil : I believe this may be of interest to you and whoever gets to integrate GitHub token scanning, see https://github.com/ewjoachim/notgithub-token-scanning

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -142,3 +142,10 @@ services:
       - "8125:8125/udp"
     volumes:
       - ./dev/notdatadog.py:/opt/warehouse/dev/notdatadog.py
+
+  notgithub:
+    image: ewjoachim/notgithub-token-scanning
+    environment:
+      NOTGITHUB_DEFAULT_URL: "http://web:8000/_/github/disclose-token"
+    ports:
+      - "8964:8000"
```
Comment (ewjoachim) [inline docker-compose.yml:151]: Picked a random port to avoid clashing.

File: docker-compose.yml [hunk 2]
```diff
@@ -142,3 +142,10 @@ services:
       - "8125:8125/udp"
     volumes:
       - ./dev/notdatadog.py:/opt/warehouse/dev/notdatadog.py
+
+  notgithub:
+    image: ewjoachim/notgithub-token-scanning
```
Comment (ewjoachim) [inline docker-compose.yml:147]: If you're curious, https://github.com/ewjoachim/notgithub-token-scanning

**Comment by greysteil** [comment]
Oh nice, thanks for the ping @ewjoachim.

**Review by di — APPROVED** [review]

**Comment by ewjoachim** [comment]
Thank you :)

**Pull request merged**